### PR TITLE
feat: miscellaneous small changes

### PR DESCRIPTION
**Pull request by grunweg**
See individual commit messages for details.

### CODE REVIEW

File: dashboard.sh [hunk 1]
```diff
@@ -62,7 +62,7 @@ gh api graphql --paginate --slurp -f query="$QUERY_AUTOMERGE" | jq '{"output": .
 
 
 # Query Github API for all pull requests that are labeled `maintainer-merge` but not `ready-to-merge` and have not been updated in 24 hours.
-QUERY_MAINTAINERMERGE=$(prepare_query "sort:updated-asc is:pr state:open label:maintainer-merge -label:ready-to-merge updated:<$yesterday")
+QUERY_MAINTAINERMERGE=$(prepare_query "sort:updated-asc is:pr state:open -label:delegated label:maintainer-merge -label:ready-to-merge updated:<$yesterday")
```
Comment (jcommelin) [inline dashboard.sh:65]: I would prefer to keep delegated PRs in the list. Because then we can ping the person who forgot about the PR.

Comment (grunweg) [reply dashboard.sh:65]: But we do have a list of "stale delegated" PR, further below...

Comment (grunweg) [reply dashboard.sh:65]: Taking a step back: the current lists have some amount of duplication, and I am not if the current partitioning is the best one. For instance, "stale delegated" also contains the one stale ready-to-merge PR. (Arguably, for that PR the ready-to-merge label should just be removed.)

Comment (grunweg) [reply dashboard.sh]: I have reverted this step for now: the discussion about duplication in these lists is a broader topic than this PR:

**Pull request merged**